src/storage/query/QueryBaseProcessor.inl

```diff
@@ -382,14 +406,33 @@ kvstore::ResultCode QueryBaseProcessor<REQ, RESP>::collectVertexProps(
                             const std::vector<PropContext>& props,
                             FilterContext* fcontext,
                             Collector* collector) {
+    auto schema = this->schemaMan_->getTagSchema(spaceId_, tagId);
     if (FLAGS_enable_vertex_cache && vertexCache_ != nullptr) {
         auto result = vertexCache_->get(std::make_pair(vId, tagId), partId);
         if (result.ok()) {
             auto v = std::move(result).value();
             auto reader = RowReader::getTagPropReader(this->schemaMan_, v, spaceId_, tagId);
-            this->collectProps(reader.get(), "", props, fcontext, collector);
-            VLOG(3) << "Hit cache for vId " << vId << ", tagId " << tagId;
-            return kvstore::ResultCode::SUCCEEDED;
+
+            // Check if the schema has TTL
+             auto retTtlOpt = getTagTTLInfo(tagId);
```

Comment:
```
        auto retTtlOpt = getTagTTLInfo(tagId);
        if (retTtlOpt.hasValue()) {
            auto ttlValue = retTtlOpt.value();
            if (checkDataExpiredForTTL(schema.get(),
                                       reader.get(),
                                       ttlValue.first,
                                       ttlValue.second)) {
                return ret;
            }
        }
```

Reply:
I originally wanted to do this, but then the log information would be incomplete.
But this code is more concise, let me do it first